### DISCUSSION TITLE
Error in mv_harmonise_data(exposure_dat, outcome_dat) :    不是所有的nexp > 1都是TRUE

### DISCUSSION THREAD

**Discussion by floor888** [MRCIEU/TwoSampleMR#566] [Q&A]
Hi,
I'm trying to harmonise data in multivariable MR but getting the following error:
```
>   mvdat <- mv_harmonise_data(exposure_dat, outcome_dat)

Error in mv_harmonise_data(exposure_dat, outcome_dat) : 
  不是所有的nexp > 1都是TRUE
```
Please help me to solve this problem！

**Comment by Cerulean25** [comment]
I've encountered the same issue as you. May I ask if you have resolved it?
